container/service.proto

```diff
@@ -24,7 +24,9 @@ service ContainerService {
   // `Delete` invokes `Container` smart contract's `Delete` method and returns
   // response immediately. After a new block is issued in sidechain, request is
   // verified by Inner Ring nodes. After one more block in sidechain, container
-  // is added into smart contract storage.
+  // is removed from smart contract storage. Only container owner has permission
+  // to do this operation. If container has NFT related attributes, then NFT
+  // owner considered as container owner.
```

Comment:
then the NFT owner is considered the current container owner.

@EliChin Please check.